ts-starter-complete-cotta/scripts/main.ts

```diff
@@ -44,8 +44,8 @@ function initializeBreakTheTerracotta() {
     player.runCommand("scoreboard players set @s score 0");
 
     let inv = player.getComponent("inventory") as EntityInventoryComponent;
-    inv.container.addItem(new ItemStack("diamond_sword"));
-    inv.container.addItem(new ItemStack("dirt", 64));
+    inv.container?.addItem(new ItemStack("diamond_sword"));
+    inv.container?.addItem(new ItemStack("dirt", 64));
```

Comment:
nit: this could use vanilla data. No need to push again if there is nothing else topush though.

Reply:
I added a user story for this case in specific. I prefer to cover cotta in a new PR. Readme file should match with the code changes and it is preferable to avoid mistakes with those changes.